### ISSUE TITLE
__slots__ = list should be detected by invalid-slots

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#775]
```
class Bad(object):
    __slots__ = list
```

The following code should emit an invalid-slots warning, but it currently doesn't do that. Interesting enough, there is a test in slots_checks.py for this, but it is disabled.
